scripts/release_scripts/run_macaron.sh

```diff
@@ -314,25 +307,24 @@ if [[ -n "${sbom_path}" ]]; then
     mounts+=("-v" "${sbom_path}:${MACARON_WORKSPACE}/sbom/${file_name}:ro")
 fi
 
-# MACARON entrypoint - Verify action argvs
-# Determine the provenance path to be mounted into ${MACARON_WORKSPACE}/provenance/${file_name}
-# This is for macaron verify action.
-if [[ -n "${arg_provenance}" ]]; then
-    provenance="${arg_provenance}"
-    err=$(check_file_exists "${provenance}" "-pr/--provenance")
+# Determine the provenance expectation path to be mounted into ${MACARON_WORKSPACE}/prov_expectations/${file_name}
+if [[ -n "${arg_prov_exp}" ]]; then
```

Comment:
I think we should also handle the case where `arg_prov_exp` is a directory.

Reply:
See commit [6a71cf8](https://github.com/oracle-samples/macaron/pull/192/commits/6a71cf8de74d95541daa6ada412d8f6e2f64661d).